File: ios/IDU2/IDU2/AppConfig.swift
import Foundation

enum AppConfig {
    // WKWebView talks to IDU directly, so authentication cookies never pass
    // through the old proxy Worker.
    static let iduBaseURL = URL(string: "https://idu.edu.pl")!

    // Browser customizations are published by `npm run publish`.
    static let contentScriptURL = URL(
        string: "https://idu-upgrade-assets.szymon-nikitin.workers.dev/content.js"
    )!
    static let stylesheetURL = URL(
        string: "https://idu-upgrade-assets.szymon-nikitin.workers.dev/styles.css"
    )!

    // Must match com.apple.security.application-groups in both target entitlements.
    static let appGroupID = "group.idu"
}

File: android/app/src/main/java/pl/edu/idu/upgrade/AppConfig.kt
package pl.edu.idu.upgrade

object AppConfig {
    const val BASE_URL = "https://idu.edu.pl"
    const val CONTENT_SCRIPT_URL = "https://idu-upgrade-assets.szymon-nikitin.workers.dev/content.js"
    const val STYLESHEET_URL = "https://idu-upgrade-assets.szymon-nikitin.workers.dev/styles.css"
    // IDU redirects the shared entry point to a tenant subdomain such as
    // s35.idu.edu.pl. Both rules are required because the wildcard does not
    // match the apex host itself.
    val allowedOrigins = setOf(
        BASE_URL,
        "https://*.idu.edu.pl",
    )
}
